### PR TITLE
reduce CPU request for job in wg-k8s-infra cluster

### PR DESCRIPTION
**Pull request by BenTheElder**
should resolve https://github.com/kubernetes/kubernetes/issues/93334

### CODE REVIEW

File: config/jobs/kubernetes/sig-testing/conformance-e2e.yaml [hunk 1]
```diff
@@ -227,7 +227,7 @@ periodics:
           # this is mostly for building kubernetes
           memory: "9000Mi"
           # during the tests more like 3-20m is used
-          cpu: 7400m
+          cpu: 7300m
```
Comment (liggitt) [inline config/jobs/kubernetes/sig-testing/conformance-e2e.yaml:230]: any chance of a comment explaining how we arrived here?

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/conformance-e2e.yaml:230]: this approximately is the remaining CPU on the node after the sidecar started using 100m and the GKE overhead.
visibility into this is poor. this was not meant to be a lasting plan.

**Comment by liggitt** [comment]
ok...

/lgtm

**Comment by BenTheElder** [comment]
I wrote up a quick comment on this.
It's not great.

**Pull request merged**